pkg/controller/servicebindingrequest/olm.go

```diff
@@ -110,15 +112,27 @@ func (o *OLM) loopCRDDescriptions(
 }
 
 // SelectCRDByGVK return a single CRD based on a given GVK.
-func (o *OLM) SelectCRDByGVK(gvk schema.GroupVersionKind) (*olmv1alpha1.CRDDescription, error) {
+func (o *OLM) SelectCRDByGVK(gvk schema.GroupVersionKind, crd *unstructured.Unstructured) (*olmv1alpha1.CRDDescription, error) {
 	log := o.logger.WithValues("Selector.GVK", gvk)
```

Comment:
@isutton would you use the name `logger` here?

Reply:
This PR isn't that small, so I prefer we add an issue to standardize this as a separate work unit. Is this fine?

Reply:
It's not about side, but about meaningful variable names. In this case, `logger` has been the one used everywhere, and we should use it here as well.